src/utils/wallet.js

```diff
@@ -316,6 +315,32 @@ const wallet = {
     store.dispatch(partiallyUpdateHistoryAndBalance({ tokensHistory, tokensBalance }));
   },
 
+  /**
+   * Fetches both history and balance for the affected tokens in updatedBalanceMap
+   *
+   * @param {HathorWallet} wallet object
+   * @param {Object} updatedBalanceMap An object containing tokens as the keys and their updated balances as values
+   *
+   * @memberof Wallet
+   * @inner
+   **/
+  async handlePartialUpdate (wallet, updatedBalanceMap) {
```

Comment:
Is it new in this PR?

Reply:
It was not added on this PR, but for some reason this method was duplicated

Removed the duplicate on 016e5ec07686aaeb5007df5d4800b5420e2ed8aa